File: AD-HOC/1808/main.py
total = 0
n = 0
s = int(input())
while s > 0:
    digit = s % 10
    if digit > 0:
        total += digit
    else:
        s //= 10
        total += 10
    s //= 10
    n += 1
print(f'{total / n:.2f}')

File: AD-HOC/1808/main.js
let line = require('fs').readFileSync('/dev/stdin', 'utf-8').trim();
let total = 0;
let n = 0;
for (let s = line.split(''); s.length; n++) {
    let digit = parseInt(s.pop());
    total += digit;
    if (digit === 0) {
        s.pop();
        total += 10;
    }
}
console.log((total / n).toFixed(2));
